packages/contracts-bedrock/test/SystemConfig.t.sol

```diff
@@ -309,7 +313,7 @@ contract SystemConfig_Setters_TestFail is SystemConfig_Initializer {
 
 contract SystemConfig_Setters_Test is SystemConfig_Initializer {
     /// @dev Tests that `setSequencer` updates the batcher hash successfully.
-    function testFuzz_setSequencer_succeeds(Types.SequencerKeys calldata sequencer) external {
+    function testFuzz_setSequencer_succeeds(Types.SequencerKeyPair calldata sequencer) external {
```

Comment:
I didn't realize you could fuzz with custom types

Reply:
I double checked, it generates unique inputs on each run.